lib/msal-core/package.json

```diff
@@ -25,12 +25,25 @@
   "engines": {
     "node": ">=0.8.0"
   },
+  "nyc": {
+    "extension": [
+      ".ts",
+      ".tsx"
+    ],
+    "include": [
+      "src"
+    ],
+    "exclude": [
+      "**/*.d.ts"
+    ],
+    "all": true  
+  },
```

Comment:
Can we instead put this in an `.nycrc` file? I'm personally not the biggest fan of putting configs for tools inside `package.json` files.

Reply:
Done.